webhook_server/tests/test_runner_handler.py

```
@@ -2547,3 +2557,194 @@ async def test_run_check_in_progress_rerun(self, runner_handler: RunnerHandler,
+            patch.object(
+                runner_handler, "run_podman_command", new=AsyncMock(return_value=(True, "ok", ""))
+            ) as mock_cmd,
+            patch.object(
+                runner_handler.check_run_handler, "is_check_run_in_progress", new=AsyncMock(return_value=False)
+            ),
+            patch.object(runner_handler.check_run_handler, "get_check_run_text", return_value="test output"),
+            patch.object(runner_handler.check_run_handler, "set_check_in_progress", new=AsyncMock()),
+            patch.object(runner_handler.check_run_handler, "set_check_success", new=AsyncMock()),
+        ):
+            mock_checkout.return_value = AsyncMock()
+            mock_checkout.return_value.__aenter__ = AsyncMock(return_value=(True, "/tmp/wt", "", ""))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/wt"

(S108)